static/third/bootstrap/css/bootstrap.css

```diff
@@ -747,8 +747,10 @@ dd {
 hr {
   margin: 20px 0;
   border: 0;
-  border-top: 1px solid #eeeeee;
-  border-bottom: 1px solid #ffffff;
+  /* border-top: 1px solid #eeeeee;
+  border-bottom: 1px solid #ffffff; */
+  border-top: 2px solid #e1e4e8;
+  border-bottom: 1px solid #e1e4e8;
```

Comment:
@manavdesai27 You should never make changes to third-party libraries (Bootstrap in this case). Plus, even if it wasn't a third-party library, the changes you're making are very generic and would affect the whole app, while it should only affect the horizontal line inside the message, that too only in light mode.

Reply:
You can probably make these changes in `rendered_markdown.css`. But then make sure to test your changes everywhere the `.rendered_markdown` class is being used, both in day-mode and night-mode.

Reply:
@garg3133 Got it, I will amend it.